### ISSUE TITLE
web client howto uses bad markup when talking about User-Agent header

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4234]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4234|
|Type|defect|
|Created|2010-01-26 22:00:27Z|

_This code sends a request which includes one custom header, I{User-Agent}._

Attachments:

* [markup.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/94c/94c2f8f600aadaf9fbb09bfa06bba8a3d232e410/aa178fcf172348e2c68a90fcc5e8df6c925b05be.patch) (696 bytes) - added by exarkun on 2010-01-27 13:50:56Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4234 4234
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1264543227000000 1264543227000000
changetime__1264611149000000 1264611149000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_2"></a>|@khorn set owner to @exarkun|
|-|-|

Looks good.  Merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [28076]) Apply markup.patch fixing incorrect markup in the web client howto

Author: exarkun
Reviewer: khorn
Fixes: [#4234](https://github.com/twisted/twisted/issues/4234)

Change markup around the string "User-Agent" from the incorrect epytext
markup to the equivalent Lore/HTML markup.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
